frontend/pages/ManageControlsPage/OSSettings/cards/DiskEncryption/components/DiskEncryptionTable/_styles.scss

```diff
@@ -12,6 +12,13 @@
   .linkToFilteredHosts__header {
     width: auto;
     max-width: 120px;
+    padding-left: 0 !important; // Removes 24px of unnecessary table scroll
+  }
+
+  .linkToFilteredHosts__cell {
+    width: auto;
+    max-width: 120px;
+    padding-left: 0 !important; // Removes 24px of unnecessary table scroll
```

Comment:
I removed 24px padding left of the filtered host link arrow because it's a lot of dead space between the last column (linux hosts) and the arrow causing unnecessary scroll

Reply:
If there's another way without using `!important`, that would be better

Reply:
I can ask @eugkuo how he feels about removing left padding on that link all together, the column left of it already has right padding so I feel like double padding is just asking for unnecessary white space scroll

Reply:
tldr: I removed the left padding to the view all host link so the user is less likely to need to scroll over to view it. I had to use `!important` which is ill-advised in frontend land to get it to apply without updating the component itself

@eugkuo lmkwyt about the double padding. If we can remove it globally, I can remove `!important`


here's a before picture: https://github.com/fleetdm/fleet/pull/25017/files#r1898061775